app/resources/v1/users_pg.js

```diff
@@ -68,59 +68,28 @@ exports.post = async function (req, res) {
         user.auth0_id = credentials.auth0_id
         user.profile_image_url = credentials.profile_image_url
         if (user.login_tokens) {
-          user.login_tokens.push(loginToken)
+          const newArray = user.login_tokens.concat(loginToken)
+          user.login_tokens = newArray
         } else {
           user.login_tokens = [loginToken]
         }
-        user.save().then(handleUpdateUser)
-      }
-    } catch (err) {
-      logger.error(err)
-      console.log(err)
-      res.status(500).json({
-        status: 500,
-        msg: 'Error finding user with Auth Twitter Sign-in.'
-      })
-    }
-  } // END function - handleAuth0TwitterSignIn
 
-  const handleTwitterSignIn = async function (credentials) {
-    try {
-      let user
-      if (credentials.auth0_id) {
-        user = await User.findOne({ where: { id: credentials.screenName } })
-      }
-      loginToken = hasPriorCall ? res.mainData.loginToken : uuid.v1()
-      if (!user) {
-        const newUserData = {
-          id: credentials.screenName,
-          _id: credentials.auth0_id.split('|')[1],
-          auth0_id: credentials.auth0_id,
-          login_tokens: [loginToken],
-          profile_image_url: credentials.profile_image_url
-        }
-        User.create(newUserData).then(handleCreateUser)
-      } else {
-        user.id = credentials.screenName
-        user.auth0_id = credentials.auth0_id
-        user.profile_image_url = credentials.profile_image_url
-        if (user.login_tokens) {
-          user.login_tokens.push(loginToken)
-        } else {
-          user.login_tokens = [loginToken]
+        try {
+          // await User.update(user, { where: { id: credentials.screenName } })
```

Comment:
Do we need to keep this commented out line? If so, please leave a comment explaining why.